app/docker/views/containers/console/exec.html

```diff
@@ -69,10 +69,28 @@
             </div>
           </div>
           <div ng-if="state !== states.disconnected">
-            <label
-              >Exec into container as <code>{{ ::formValues.user || 'default user' }}</code> using command
-              <code>{{ formValues.isCustomCommand ? formValues.customCommand : formValues.command }}</code></label
-            >
+            <label class="control-label text-left"
+              >Exec &nbsp;into container as <code>{{ ::formValues.user || 'default user' }}</code> using command
+              <code>{{ formValues.isCustomCommand ? formValues.customCommand : formValues.command }}</code>
+              <portainer-tooltip
```

Comment:
use the same tooltip as in app/react/kubernetes/applications/ConsoleView/ConsoleView.tsx (just export it and import to the controller, then put it on the scope